### PR TITLE
Change default shingle size for HC detector to 4

### PR DESCRIPTION
**Pull request by yizheliu-amazon**
*Issue #, if available:*

*Description of changes:*
Change default shingle size for HC detector to 4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/createDetector/containers/utils/detectorToFormik.ts [hunk 1]
```diff
@@ -41,6 +41,6 @@ export function detectorToFormik(ad: Detector): ADFormikValues {
     timeField: ad.timeField,
     detectionInterval: get(ad, 'detectionInterval.period.interval', 10),
     windowDelay: get(ad, 'windowDelay.period.interval', 0),
-    shingleSize: get(ad, 'shingleSize', SHINGLE_SIZE),
+    shingleSize: get(ad, 'shingleSize', SINGLE_ENTITY_SHINGLE_SIZE),
```
Comment (ohltyler) [inline public/pages/createDetector/containers/utils/detectorToFormik.ts:44]: Can we use the `getShingleSizeFromObject` helper fn you made, in the case that a HC detector is passed here? Could consider if there is a `categoryField` then `isHCDetector` would be `true`.

File: public/utils/constants.ts [hunk 2]
```diff
@@ -66,7 +66,9 @@ export const MAX_FEATURE_NAME_SIZE = 64;
 export const NAME_REGEX = RegExp('^[a-zA-Z0-9._-]+$');
 
 //https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/ad/settings/AnomalyDetectorSettings.java#L186
-export const SHINGLE_SIZE = 8;
```
Comment (ohltyler) [inline public/utils/constants.ts:69]: can you confirm all instances of `SHINGLE_SIZE` have been changed to `MULTI_ENTITY_SHINGLE_SIZE`?

Comment (yizheliu-amazon) [reply public/utils/constants.ts:69]: confirmed.

**Review by ohltyler — APPROVED** [review]
Thanks for the change!

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**